rosidl_adapter/rosidl_adapter/msg/__init__.py

```diff
@@ -61,10 +61,8 @@ def convert_msg_to_idl(package_dir, package_name, input_file, output_dir):
 
 def to_idl_literal(idl_type, value):
     if idl_type[-1] == ']' or idl_type.startswith('sequence<'):
-        elements = [repr(v) for v in value]
-        while len(elements) < 2:
-            elements.append('')
```

Comment:
@ivanpauno just to double check, is it guaranteed that `elements` will never be empty? I'd be curious about the original code if not though.

Reply:
> just to double check, is it guaranteed that elements will never be empty?

Maybe value can be `[]`, I think that can be an acceptable default value for an array.
IDK what that `while` is doing though, it doesn't seem to make sense.

Reply:
> IDK what that while is doing though, it doesn't seem to make sense.

IIUC it was ensuring you get `(value,)` if `len(elements) == 1` but you'd get `(,)` if `len(elements) == 0`, which is nonsense. 

> Maybe value can be [], I think that can be an acceptable default value for an array.

Maybe? Anyhow, your code will result in `()`, which is a bit more sensible if allowed.